doc/sphinx-guides/source/installation/localcontexts.rst

```diff
@@ -0,0 +1,31 @@
+LocalContexts Integration
+=========================
+
+.. contents:: |toctitle|
+  :local:
+
+`Local Contexts <https://localcontexts.org/>`_ is a global initiative that supports Indigenous communities in the management and sharing of their cultural heritage and data.
+The `Local Contexts Hub <https://localcontextshub.org/>`_ is a platform that enables the creation and application of Traditional Knowledge (TK) and Biocultural (BC) Labels and Notices.
+These labels and notices help to communicate the cultural context and appropriate use of Indigenous data and cultural heritage materials.
+
+Dataverse supports integration with the Local Contexts Hub so that Labels and Notices associated with a dataset can be displayed on the dataset page:
+
+.. figure:: ./img/LCDemo.png
+   :alt: Dataset Page showing Local Contexts integration with Dataverse Software
+
+Configuration
+-------------
+
+There are several steps to LocalContexts integration.
+
+First, you need to configure the LOCALCONTEXTS_URL and LOCALCONTEXTS_API_KEY as described in the :ref:`localcontexts` section of the Configuration Guide.
+API Keys are available to Local Contexts Integration Partners - see https://localcontexts.org/hub-agreements/integration-partners/ for details.
+
+Next, you should add the Local Contexts metadatablock and configure the associated external vocabulary script.
+The metadatablock contains one field allowing Dataverse to store the URL of an associated Local Contexts Hub project.
```

Comment:
We definitely want the metadata block in the other repo? I'm fine with it but having it in the main list might be an extra way to advertise TK Labels. Plus it could be used in tests, etc.

Reply:
I added a reference to the block and this documentation in the experimental metadata block section in the appendix. I'm not sure that moving it and managing it in the main repo helps as it probably doesn't make sense to use it without the external vocab script.

Reply:
The change you made looks good. Thanks!